casestudies/load-testing/README.md

```diff
@@ -0,0 +1,70 @@
+# ACK Performance at Scale
+
+In this usecase, we would deploy 10K SNS Topics, 20K SQS Queues and 20K SNS Subscriptions i.e 50K resources at a time using SQS and SNS ACK Controllers to examine performance of ACK and other metrics.
```

Comment:
Can you write a pre intro on what this is a about, something like:
```markdown
This document presents a comprehensive analysis of the performance and scalability of AWS
Controllers for Kubernetes (ACK) when deploying a large number of resources. The purpose of this
study is to examine how ACK controllers handle the creation and management of 50k resources, including
SNS topics and SQS queues
```

Reply:
Added an intro in the new commit